torch/csrc/jit/script/compiler.cpp

```diff
@@ -1463,22 +1524,30 @@ struct to_ir {
               tree->range(), entry->asValue(starred.range(), method)));
         }
       } else {
-        values.push_back(NamedValue(
-            tree->range(), emitExpr(Expr(tree))));
+        auto sugared_expr = emitSugaredExpr(Expr(tree), 1);
+        // TODO: non-sketchy check
+        if (collect_blocks && sugared_expr->kind() == "module") {
```

Comment:
`kind` is for debugging. This doesn't work correctly because lots of SugaredValues are callable but not a module.